### PR TITLE
672 arrow filter

### LINKED ISSUE #672: Arrow filter
Add arrow filter to map filter menu

### PR DESCRIPTION
**Pull request by Divinare**
Closes #672

Notify: I made this branch on top of pr #670 branch. So first merge 670, then merge this

### CODE REVIEW

File: src/components/map/mapControls/MapLayersControl.tsx [hunk 1]
```diff
@@ -80,32 +84,33 @@ class MapLayersControl extends React.Component
...
                             onClick={this.toggleMapLayerVisibility(MapLayer.node)}
                             checked={NetworkStore.isMapLayerVisible(MapLayer.node)}
-                            text={'Näytä alueen solmut'}
+                            text={'Alueen solmut'}
                         />
                     </div>
                     <div className={s.checkboxContainer}>
                         <Checkbox
-                            onClick={this.toggleMapLayerVisibility(MapLayer.nodeWithoutLink)}
+                            onClick={this.toggleMapLayerVisibility(MapLayer.nodeWithoutLink)} // tslint:disable
```
Comment (kronken) [inline src/components/map/mapControls/MapLayersControl.tsx]: Why is there an tslint here, and this tslint will disable tslinting for the whole file

**Comment by kronken** [comment]
No real review done yet, waiting for the first task to be merged. However, found a tslint that seemed weird, so I added a comment

File: src/components/map/mapControls/MapLayersControl.tsx [hunk 2]
```diff
@@ -80,32 +84,33 @@ class MapLayersControl extends React.Component
                     />
                     <NetworkDateControl selectDate={this.selectDate}/>
                     <div className={s.sectionDivider} />
+                    <div className={s.inputTitle}>GEOMETRIAT</div>
                     <div className={s.checkboxContainer}>
                         <Checkbox
                             onClick={this.toggleMapLayerVisibility(MapLayer.link)}
                             checked={NetworkStore.isMapLayerVisible(MapLayer.link)}
-                            text={'Näytä alueen linkit'}
+                            text={'Alueen linkit'}
```
Comment (kronken) [inline src/components/map/mapControls/MapLayersControl.tsx]: Set
text={'Alueen linkit'}
=>
text='Alueen linkit'
on all similar places in this file, there are plenty

Comment (Divinare) [reply src/components/map/mapControls/MapLayersControl.tsx]: Refactored. I guess that is our convention from now on through the app

**Pull request merged**